apps/webapp/src/app/(Magic-Link)/magic-link/page.tsx

```
@@ -0,0 +1,614 @@
+              ))}
+            </SelectContent>
+          </Select>
+        </div>
+
+        <div className="space-y-2 max-h-96 overflow-y-auto">
+          {filteredProviders.map((provider) => (
+            <div
+              key={`${provider.name}-${provider.vertical}`}
+              className="flex items-center p-2 hover:bg-gray-100 cursor-pointer rounded"
+              onClick={() => handleProviderSelect(provider)}
+            >
```

Biome

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

Actions triggered using mouse events should have corresponding keyboard events to account for keyboard-only navigation.

(lint/a11y/useKeyWithClickEvents)